*** ./smali/com/android/internal/telephony/gsm/SpnOverride.smali	2015-12-08 00:15:31.959229906 +0800
--- /data/patchrom_jb/s6818/temp/new_smali/framework.jar.out/./smali/com/android/internal/telephony/gsm/SpnOverride.smali	2015-12-08 00:13:39.155232281 +0800
***************
*** 271,273 ****
--- 271,286 ----
  
      return-object v0
  .end method
+ 
+ .method getSpn(Ljava/lang/String;Ljava/lang/String;)Ljava/lang/String;
+     .locals 1
+     .parameter "carrier"
+     .parameter "spn"
+ 
+     .prologue
+     invoke-virtual {p0, p1}, Lcom/android/internal/telephony/gsm/SpnOverride;->getSpn(Ljava/lang/String;)Ljava/lang/String;
+ 
+     move-result-object v0
+ 
+     return-object v0
+ .end method
