*** ./smali/com/android/server/AlarmManagerService.smali	2015-12-08 00:15:46.603229598 +0800
--- /data/patchrom_jb/s6818/temp/new_smali/services.jar.out/./smali/com/android/server/AlarmManagerService.smali	2015-12-08 00:13:53.439231980 +0800
***************
*** 2055,2061 ****
  .end method
  
  .method public setRepeating(IJJLandroid/app/PendingIntent;)V
!     .locals 4
      .parameter "type"
      .parameter "triggerAtTime"
      .parameter "interval"
--- 2055,2061 ----
  .end method
  
  .method public setRepeating(IJJLandroid/app/PendingIntent;)V
!     .locals 7
      .parameter "type"
      .parameter "triggerAtTime"
      .parameter "interval"
***************
*** 2074,2079 ****
--- 2074,2099 ----
      return-void
  
      :cond_0
+     iget-object v0, p0, Lcom/android/server/AlarmManagerService;->mContext:Landroid/content/Context;
+ 
+     move v1, p1
+ 
+     move-wide v2, p2
+ 
+     move-wide v4, p4
+ 
+     invoke-static/range {v0 .. v5}, Lcom/android/server/ExtraAlarmManagerService;->alignAlarm(Landroid/content/Context;IJJ)[J
+ 
+     move-result-object v2
+ 
+     const/4 v0, 0x0
+ 
+     aget-wide p2, v2, v0
+ 
+     const/4 v0, 0x1
+ 
+     aget-wide p4, v2, v0
+ 
      iget-object v3, p0, Lcom/android/server/AlarmManagerService;->mLock:Ljava/lang/Object;
  
      monitor-enter v3
