*** ./smali/com/android/internal/view/ActionBarPolicy.smali	2015-12-08 00:15:21.607230124 +0800
--- /data/patchrom_jb/s6818/temp/new_smali/framework.jar.out/./smali/com/android/internal/view/ActionBarPolicy.smali	2015-12-08 00:13:29.195232490 +0800
***************
*** 227,232 ****
--- 227,236 ----
      .locals 1
  
      .prologue
+     const/4 v0, 0x1
+     
+     return v0
+     
      iget-object v0, p0, Lcom/android/internal/view/ActionBarPolicy;->mContext:Landroid/content/Context;
  
      invoke-static {v0}, Landroid/view/ViewConfiguration;->get(Landroid/content/Context;)Landroid/view/ViewConfiguration;
